site-graph/cache/cache.go

```diff
@@ -0,0 +1,10 @@
+package cache
+
+type Cache interface {
```

Comment:
imho - too early to create package with just interface in it - if you have only one its implementation. Better to create this interface near to actual usage - in http package. So in main you create cachemap and NewServer accepts cache interface